### ISSUE TITLE
Prompt with default value and validation is broken

### ISSUE THREAD

**Issue by hardcodet** [aritchie/userdialogs#362]
- [x] Android
(didn't test others)

## Version of OS(s) listed above with issue
XF 2.3.4, Android 6

## Version of ACR UserDialogs Library
6.3.10

If I initialize a prompt with a default value and a validation routine, the dialog is being displayed with a disabled OK button, because OnTextChanged is only invoked after the user makes an edit, so from a usability perspective, those two features are mutually exclusive. Is there a workaround to this?

This config presents a valid email address with a disabled OK button:

          var cfg = new PromptConfig
            {
                Text = "foo@bar.com",
                OnTextChanged = t => t.IsValid = Validator.IsEmailAddress(t.Value)
            };

**Comment by aritchie** [comment]
Please use the template that is filled in automatically for you when you file a defect/feature request.  I can't tell what platform this is on and there could very well be a defect on one specific platform here.  I will likely be closing defects filed like this in the future without warning!

Secondly, I do run the validation just before displaying the UI, if the default value matches a good value, it should be returning true.

**Comment by hardcodet** [comment]
Updated ticket, sorry for ignoring the template. I just skipped the default value and wanted to quickly let you know. As far as the repro goes and the validation running, this is not what I'm seeing. I could just add an arbitrary character, and the OK button got enabled, remove it again and it stayed enabled.

Btw, minor thing: Cursor at the end of the default value instead of the beginning would be small UX improvement.
